server/index.js

```diff
@@ -0,0 +1,48 @@
+/* eslint-disable no-console */
+const express = require('express');
+const cors = require('cors');
+const { Pool } = require('pg');
+require('newrelic');
```

Comment:
important to require 'newrelic' at the top of the file. It can obscure the requests going through the dashboard if it's not in the first line. Request will be called anonymous. May cause other slowdowns too.